content/articles/beginner's-guide-to-lottie-animations-in-android-using-kotlin/index.md

```diff
@@ -0,0 +1,380 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: beginner's-guide-to-lottie-animations-in-android-using-kotlin
+title: Beginner's guide to Lottie animations in Android using Kotlin 
+description: This article explains how to use Lottie which is an open source animation library in Android applications using Kotlin. Lottie is beautiful and helps you achieve a clean UI, therefore achieving a good user experience.
+author: carol-musyoka
+date: 2021-05-31T00:00:00-10:30
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/beginner's-guide-to-lottie-animations-in-android-using-kotlin/hero.jpg
+    alt: Beginner's guide to Lottie animations in Android using Kotlin 
+---
+
+### Introduction
+Designing animations can be tedious and technical for most developers. **Lottie** is an open-source animation library developed by *Airbnb*. It renders Adobe After Effects with BodyMovin. **BodyMovin** is an Adobe After Effects plugin that exports animations to a JSON file. Lottie makes it easy to work with animations without downloading heavy animation files.
+Lottie helps developers achieve a clean UI, therefore achieving a good user experience. Lottie is easy to use, and the files can be accessed on their [site](https://lottiefiles.com/).
+Lottie is supported on mobile (Android, iOS, and React Native), web, and desktop applications.
+
+### Why use Lottie
+- Simple UI elements - Lottie is light and the JSON files exported are small-sized. 
+- Dynamic scenes - You can tell a narrative across several shots, making it interactive.
+- Alter animations to your liking - The animations offered on Lottie can be altered from speed to color to progress hence meeting the developer's need.
+- Improved user experience.
+- Pairs well across all mobile devices(in our case), even slow-performing ones, since it is lightweight.
+- The animations also have a clear resolution.
+
+This article will share how to implement Lottie animations in onboarding screens using Kotlin. Since the project is a bit wide, we shall only go through all the basics of Lottie animations on the app. 
+
+### Prerequisites
```

Comment:
Hey @carolinemusyoka . I don't think this is the only prerequisite needed. What about having Android Studio? I think someone also needs a little bit of experience in developing android applications in order to follow this article.